File: algorithms/neetcode/algorithms/algorithms/src/main/java/com/mycompany/leetcode/grind75/week1/ValidAnagram/ValidAnagramKshatriyas.java
package com.mycompany.leetcode.grind75.week1.ValidAnagram;

import java.util.Arrays;

public class ValidAnagramKshatriyas {
    public boolean isAnagram(String s, String t) {
        char[] sChars = s.toCharArray();
        char[] tChars = t.toCharArray();
        Arrays.sort(sChars);
        Arrays.sort(tChars);

        return Arrays.equals(sChars, tChars);
    }

    public static void main(String[] args) {
        ValidAnagramKshatriyas obj = new ValidAnagramKshatriyas();
        String s = "anagram", t = "nagaram";
        System.out.println(obj.isAnagram(s, t));
    }
}

File: algorithms/neetcode/algorithms/algorithms/src/main/java/com/mycompany/leetcode/grind75/week1/ValidAnagram/valid-anagram-kshatriyas.cc

#include <iostream>
#include <algorithm>

using namespace std;

class Solution{
  public:
    bool isAnagram(string s, string t) {
      sort(s.begin(), s.end());
      sort(t.begin(), t.end());
      return s == t;
    }
};

int main(){
  Solution obj;;
  string s = "anagram", t = "nagaram";
  std::cout << obj.isAnagram(s, t);
}
